.github/workflows/e2e-nvidia-a10g-x1.yml

```diff
@@ -116,18 +117,41 @@ jobs:
 
       - name: Install ilab
         run: |
-          export PATH="/home/ec2-user/.local/bin:/usr/local/cuda/bin:$PATH"
-          python3.11 -m venv --upgrade-deps venv
+          export CUDA_HOME="/usr/local/cuda"
+          export LD_LIBRRY_PATH="$LD_LIBRARY_PATH:/usr/local/cuda/lib64:/usr/local/cuda/extras/CUPTI/lib64"
+          export PATH="$PATH:$CUDA_HOME/bin"
+          python3.11 -m venv venv
           . venv/bin/activate
+          nvidia-smi
           sed 's/\[.*\]//' requirements.txt > constraints.txt
           python3.11 -m pip cache remove llama_cpp_python
           CMAKE_ARGS="-DLLAMA_CUBLAS=on" python3.11 -m pip install --force-reinstall --no-binary llama_cpp_python -c constraints.txt llama_cpp_python
           python3.11 -m pip install bitsandbytes
+
+          # TODO This should be added to instructlab-training
+          python3.11 -m pip install packaging wheel
```

Comment:
It looks like this was (partially?) resolved here -- https://github.com/instructlab/training/pull/123. -- it added wheel, but not packaging ...

What's the status on this? It would be great to either have this fixed, or an open issue tracking it

Reply:
@Maxusmusti I tried removing the packaging dep, but it seems the tests still fail. I can remove wheel here but I think packaging is still necessary

Reply:
then it needs to be added to the training repo, right?

Reply:
I think so

Reply:
It is already in the training repo requirements.txt though, so something odd going on here: https://github.com/instructlab/training/blob/main/requirements.txt#L1

Reply:
@russellb is there a different version needed here?

Reply:
based on that, it looks like the line can be removed. If it fails, can you share what it looks like @cdoern ?

Reply:
or if you want to just file an issue to follow up on ensuring this is fixed for good and gets removed from this file, that's fine too

Reply:
https://github.com/instructlab/instructlab/issues/1655